Content.Client/Shuttles/UI/ShuttleConsoleWindow.xaml.cs

```diff
@@ -87,6 +92,22 @@ private void OnUndockPressed(BaseButton.ButtonEventArgs args)
         UndockPressed?.Invoke(DockingScreen.ViewedDock.Value);
     }
 
+    private void OnThrustLimiterSliderChanged(Range range)
+    {
+        SetThrustLimit(range.Value);
+        ThrustLimited?.Invoke(range.Value);
+    }
+
+    public void SetThrustLimit(float limit)
+    {
+        if (MathHelper.CloseTo(limit, ThrustLimiter.Value))
+            return;
+
+        ThrustLimiterLabel.Text = Loc.GetString("shuttle-console-thrust-limit-label", ("limit", limit));
+        if(!ThrustLimiter.Grabbed)
+            ThrustLimiter.SetValueWithoutEvent(limit);
+    }
+
     public void SetMatrix(EntityCoordinates? coordinates, Angle? angle)
```

Comment:
Move this stuff to ShuttleNavControl which is the new equivalent, maybe ShuttleDockControl as well.